src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/eventDetailsBuilder.hpp

```diff
@@ -267,6 +267,35 @@ class TEventDetailsBuilder final : public AbstractHandler<std::shared_ptr<TScanC
                                 .at("adp");
                         ecsData["vulnerability"]["under_evaluation"] = isUnderEvaluation();
 
+                        if (const auto it = context->m_matchConditions.find(cve);
+                            it != context->m_matchConditions.end())
+                        {
+                            if (it->second.condition == MatchRuleCondition::LessThanOrEqual)
+                            {
+                                ecsData["vulnerability"]["scanner"]["condition"] =
+                                    "Package less than or equal to " + it->second.version;
+                            }
+                            else if (it->second.condition == MatchRuleCondition::LessThan)
+                            {
+                                ecsData["vulnerability"]["scanner"]["condition"] =
+                                    "Package less than " + it->second.version;
+                            }
+                            else if (it->second.condition == MatchRuleCondition::DefaultStatus)
+                            {
+                                ecsData["vulnerability"]["scanner"]["condition"] = "Package default status";
+                            }
+                            else if (it->second.condition == MatchRuleCondition::Equal)
+                            {
+                                ecsData["vulnerability"]["scanner"]["condition"] =
+                                    "Package equal to " + it->second.version;
+                            }
+                            else
+                            {
+                                ecsData["vulnerability"]["scanner"]["condition"] = "Unknown";
```

Comment:
After evaluating the code, this situation doesn't seem possible.
Writing unknown doesn't harm, and can give us more information in the future in case we add more conditions without modify this.